Gemfile

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
More expandability for future potential additions if we just leave it like though. Won't hurt much as it stands.